src/list/ListItem.js

```diff
@@ -16,12 +16,23 @@ import Icon from '../icons/Icon';
 import Text from '../text/Text';
 import ButtonGroup from '../buttons/ButtonGroup';
 import Input from '../input/Input';
-import Divider from '../divider/Divider';
+import { dividerColor } from '../divider/Divider';
```

Comment:
Prefer if you added this in the [colors.js](https://github.com/react-native-training/react-native-elements/blob/master/src/config/colors.js) file since it's used between two components. Other than that everything is 💯